src/nsrt_learning.py

```diff
@@ -220,64 +207,65 @@ def _get_initial_nids(segments: Sequence[Segment], verbose: bool
                 preconditions = lifted_atoms
             else:
                 preconditions &= lifted_atoms
-        nid.op = STRIPSOperator(
-            nid.op.name, nid.op.parameters, preconditions, nid.op.add_effects,
-            nid.op.delete_effects, nid.op.side_predicates)
+        pnad.op = STRIPSOperator(
```

Comment:
it's pretty weird to create an initial STRIPSOperator before this and then destroy it now. can we avoid this somehow, and create the STRIPSOperator just once?

Reply:
for example, keep the effects in the PNAD, and then add preconditions, and then call `make_strips_operator` on the PNAD or something?